<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPushDialog</class>
 <widget class="QDialog" name="CPushDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Push Values to Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Origin map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c_origin_layer">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Origin attribute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c_origin_attribute">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Push values to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_layer_selector"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>If destination object intersects more than
 one object in origin map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_function">
       <item>
        <property name="text">
         <string>Take maximum attribute value</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Take minimum attribute value</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Take average attribute value</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Take total of attribute values</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="c_count_intersections">
     <property name="text">
      <string>Record object intersection count</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="c_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_ok</sender>
   <signal>clicked()</signal>
   <receiver>CPushDialog</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CPushDialog</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOK()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
